board.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"/>
+    <title>chessboardjs Example #1000 - Empty Board</title>
+    <base href="../"/>
+    <link rel="stylesheet" href="https://unpkg.com/@chrisoakman/chessboardjs@1.0.0/dist/chessboard-1.0.0.min.css"
+          integrity="sha384-q94+BZtLrkL1/ohfjR8c6L+A6qzNH9R2hBLwyoAfu3i/WCvQjzL2RQJ3uNHDISdU" crossorigin="anonymous">
+</head>
+<body>
+
+<p>Ably chess viewer</p>
+
+<div id="myBoard" style="width: 400px"></div>
+<p>
+<div id="thismove"></div>
+</p>
+
+<script src="https://cdn.ably.com/lib/ably.min-1.js"></script>
+<script src="https://chessboardjs.com/js/jquery-3.4.1.min.js"></script>
+<script src="https://unpkg.com/@chrisoakman/chessboardjs@1.0.0/dist/chessboard-1.0.0.min.js"
+        integrity="sha384-8Vi8VHwn3vjQ9eUHUxex3JSN/NFqUg3QbPyX8kWyb93+8AC/pPWTzj+nHtbC5bxD"
+        crossorigin="anonymous"></script>
+<script>
+    var board = Chessboard('myBoard')
+
+    async function displayGame() {
+        // Use a very restricted Ably API Key which only allows the user to subscribe to messages on the game1
+        // channel in the chessdemo app.
+        const ably_key = '6Z_qIg.wS-GtA:DTq-7TiASaOOPat9IJ7zVm-jmuohkXW8ujbGs1zPthU';
```

Comment:
Not elegant, but a practical precautionary measure.
Prompt for the KEY, or use the session value.

Uses the Session, so the key is not seen in the wild.

```suggestion
        const ably_key = sessionStorage.ABLY_KEY || prompt("Enter your key");
        if(/^[a-z0-9_\-.:]{32,64}$/i.test(ably_key)) sessionStorage.ABLY_KEY = ably_key;
```

Reply:
cool!
What happens if the key gets revoked?
Will the bad key be stuck in sessionStorage for ever?

Reply:
umm yes and no. yes it is stuck and no, because it is the browser session, so it is forgotten when the browser restarts. and so the user needs to re-validate anyway.